include/kernels/ElectronEnergyLossFromIonization.h

```diff
@@ -12,6 +12,11 @@
 
 #include "ADKernel.h"
 
+/**
+ *  Electron energy loss term for inelastic ionization collisions
+ *  using Townsend coefficients, the energy lost in Volts in a single
+ *  ionization collision (Densities must be in logarithmic form)
```

Comment:
Yes, this is the correct words, since it reference energy lost in a single ionization collision. If reference, this is the wording from Alex thesis and Lieberman uses similar nomenclature.